lib/util.ml

```diff
@@ -22,12 +22,15 @@ module Result = struct
     let ( >>! ) r f =
       match r with
       | Ok x -> f x
-      | Error (`Msg e) ->
-          Printf.eprintf "[mdx] Fatal error: %s\n" e;
+      | Error l ->
+          List.iter
+            (function `Msg m -> Printf.eprintf "[mdx] Fatal error: %s\n" m)
+            l;
```

Comment:
Purely stylistic but I find it somewhat weird to print a fatal error and keep going, maybe it would make sense to print them as errors and finally conclude that these are fatal and the program will exit?

Or maybe that's just me overthinking semantics.

Reply:
That's what it's doing, the `1` coming after is the exit code.

Reply:
Looking at how this is used by mdx_test and the dune-gen command, it's probably better to move the exit in here as well actually!

Reply:
After a bit of testing, it seems to work well without any change! We can eventually do this in a separate PR if the need arise!